src/server/user_handler.hpp

```diff
@@ -124,15 +124,15 @@ class user_handler {
 		void init_mailer(const config &c);
 
 		/** Create a random string of digits for password encryption. */
-		std::string create_salt(int length = 8);
-		std::string create_secure_salt();
+		std::string create_unsecure_nonce(int length = 8);
```

Comment:
The right word is "insecure".

Reply:
Nope. 'unsecure' is correct. `(especially of a computer system) not protected against attack or other criminal activity. "an unsecure WiFi network"`

Reply:
Pretty sure "insecure" is actually correct, but neither is downright wrong so it doesn't matter that much.